### PR TITLE
Largetable benchmark for Angular 2 (Dart and JS)

### PR DESCRIPTION
**Pull request by jeffbcross**

### CODE REVIEW

**Comment by jeffbcross** [comment]
Hey @tbosch mind taking a look at this?

File: protractor-dart2js.conf.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ config.exclude.push(
   // TODO: remove this line when largetable dart has been added
   'dist/js/cjs/benchmarks_external/e2e_test/largetable_perf.js',
   'dist/js/cjs/benchmarks_external/e2e_test/polymer_tree_perf.js'
+  // 'dist/js/cjs/benchmarks_external/e2e_test/largetable_perf.js'
```
Comment (vicb) [inline protractor-dart2js.conf.js]: remove ?

Comment (jeffbcross) [inline protractor-dart2js.conf.js]: Yep, will do! Thanks

**Comment by jeffbcross** [comment]
Rebased and pushed branch. Hopefully no flakes this time.

**Comment by jeffbcross** [comment]
Got verbal LGTM from @rkirov and merged!

**Pull request closed without merging**